style.css

```diff
@@ -0,0 +1,66 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+}
+
+.greetings {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+}
```

Comment:
- [ ] Kindly make sure to use the correct typography (like `fonts-family`) on this text in order to match the provided Figma design.

  - ### YOUR DESIGN
  
   ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/87197412/6d2c976a-bbef-492a-a9f4-2af864c441f3)

  - ### FIGMA DESIGN
  
   ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/87197412/07222028-b6e6-48ac-9e8a-9f59fa25e663)